web/src/features/auth/AuthWrapper.tsx

```diff
@@ -1,15 +1,19 @@
 import React, { useEffect } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
-import { authenticate } from 'features/auth/slices/authenticationSlice'
+import { authenticate, testAuthenticate } from 'features/auth/slices/authenticationSlice'
 import axios from 'api/axios'
 import { AppThunk } from 'app/store'
 import { selectToken, selectAuthentication } from 'app/rootReducer'
+import { TEST_AUTH } from 'utils/env'
 
 interface Props {
-  shouldAuthenticate: boolean
   children: React.ReactElement
 }
 
+// Used for test authentication
```

Comment:
A final thought (repeat thought?) - we could load the token from environment variables. That way we're a) not throwing your token out to the world b) not storing dumb test stuff in our main code.

Reply:
It DOES mean that depending on your configuration you'll get different test results. That's a bad.....